src/components/QuestionTypes/SalaryType.vue

```diff
@@ -51,7 +71,20 @@ export default {
   },
   methods: {
     isValid() {
-      return this.question.checkbox || this.dataValue;
+      if (!this.question.required) {
+        return true;
+      }
+      if (this.question.checkbox) {
+        return true;
+      }
+
+      if (
```

Comment:
As a follow up on the mask component, I tried fixing this myself by removing the condition on isValid() on line 81 in Salary.vue. This worked to allow the user to input any character, but it somehow removed the v-money effects with thousand separators. I'd be curious to see how you solve this problem!